src/hooks/journals/useJournalData.ts

```diff
@@ -81,8 +95,16 @@ const useJournalsForCollection = (collectionName: string | undefined) => {
             onError: async (error) => {
                 incAttempts();
 
-                if (session && shouldRefreshToken(`${error}`)) {
-                    await refreshAuthToken();
+                if (
+                    session &&
+                    shouldRefreshToken(`${error}`) &&
+                    collectionName
+                ) {
+                    await refreshAuthToken(
+                        session.access_token,
+                        collectionName,
```

Comment:
This is fine for now... but we probably don't want to use the `collectionName` from the outer scope. Rather I _think_ we can read it from the `swr` config that should be getting passed in to the `onError`.